spec/lib/integrations/pivotal_tracker_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+require 'integrations'
+
+describe Integrations::PivotalTracker do
+  describe '#send_event' do
+    let(:event_name) { "run_completion" }
+    let(:payload) do
+      {
+        run: {
+          id: 9,
+          status: "failed",
+          description: "rainforest run",
+          time_to_finish: 750
+        },
+        frontend_url: "http://www.rainforestqa.com/",
+        failed_tests: [
+          {
+            id: "20",
+            name: "Always fails",
+            url: "http://www.rainforestqa.com/"
+          }
+        ]
+      }
+    end
+    let(:settings) do
+      {
+        project_id: 1141528,
+        auth_token: "8537292c903ca580bcd10b800709a136",
+      }
+    end
+    let(:expected_message) { "Your Rainforest Run <http://www.rainforestqa.com/ | Run #9: rainforest run> failed. Time to finish: 12 minutes 30 seconds" }
```

Comment:
Possibly yeah, either way works for me as this is unreleased code anyways for now.